org.yakindu.sct.examples.basic.eventbuffers/metadata.json

```diff
@@ -6,7 +6,7 @@
 	"previewImage": "lights_thumb.jpg",
 	"category": [
 		"basic",
-		"modeling"
+		"execution-variants"
 	],
 	"rank":30
```

Comment:
Let's re-order the execution variant examples like this:
* Event-based vs Cycle-based
* Event Buffering
* Super Steps
* Parent-first vs Child-first